src/components/_common/chatbot/ChatbotLinkPanel.module.scss

```
@@ -3,7 +3,7 @@
 $chatPointerSize: 1.5rem;
 
 .chat {
-    $bgColor: common.$a-blue-50;
+    $bgColor: var(--a-white);
```

kunne ikke den brukt samme syntax som før?
```suggestion
    $bgColor: common.$a-white;
```